cmd/lakectl/cmd/tag_list.go

```diff
@@ -11,14 +12,14 @@ import (
 var tagListCmd = &cobra.Command{
 	Use:               "list <repository uri>",
 	Short:             "List tags in a repository",
-	Example:           "lakectl tag list lakefs://<repository>",
+	Example:           fmt.Sprintf("lakectl tag list %s", myRepoExample),
```

Comment:
1. Think it is too much, think the previous code is more readable and the chance that we will need to update this constant is close to zero. It is placeholder, not example repository. 
2. Constant - prefer compile time concatenation
```suggestion
	Example:           "lakectl tag list " + myRepoExample,
```

Reply:
It's not about readability - it's about consistency throughout the commands that was badly missing